src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -40,10 +40,16 @@
             <f:textbox/>
         </f:entry>
 
-        <f:optionalBlock name="iamge" title="Use Docker driver" inline="true" checked="${instance.image != null}">
+        <f:optionalBlock name="docker" title="Use Docker driver" inline="true" checked="${instance.image != null}">
             <f:entry title="Image">
                 <f:textbox name="image" field="image"/>
             </f:entry>
+            <f:entry title="Username">
+                <f:textbox name="username" field="username" default="none"/>
```

Comment:
Any particular reason you are settings the default to "none"? Otherwise I think we can just use the empty string? Although unlikely people may confuse the username with none.